spec/schema.json

```diff
@@ -531,6 +555,230 @@
         "start_date"
       ]
     },
+    "TrustData": {
+      "type": "object",
+      "properties": {
+        "trust_id": {
+          "type": "string"
+        },
+        "trust_name": {
+          "type": "string"
+        },
+        "creation_date": {
+          "type": "string",
+          "format": "date"
+        },
+        "unable_to_obtain_all_trust_info": {
+          "type": "boolean"
+        },
+        "INDIVIDUAL": {
```

Comment:
Just wondering why capitals have been used for these field names and also why names are not plural, e.g. 'individuals'?

Reply:
This has been taken from the architect's specification which can be found here: https://companieshouse.atlassian.net/wiki/spaces/~mmoore/pages/3787587694/ROE+Trust+JSON+Data+Scenarios

Reply:
Ok. Seems odd. Did you query this with @chrism84 or @mikemoore24 ? Suggest doing so but this can always be changed later if they agree it's wrong (of course, it might not be!).